src/hooks/useReportIsArchived.ts

```diff
@@ -0,0 +1,12 @@
+import {isArchivedReport} from '@libs/ReportUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import useOnyx from './useOnyx';
+
+function useReportIsArchived(reportID?: string): boolean {
+    const [reportNameValuePairs] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${reportID ?? CONST.DEFAULT_NUMBER_ID}`, {canBeMissing: true});
```

Comment:
I'm asking about this lint error in [Slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1744990665238199). I believe I am doing it properly...